pkg/codegen/python/gen.go

```diff
@@ -2454,14 +2464,37 @@ func (mod *modContext) genInitDocstring(w io.Writer, res *schema.Resource, resou
 	fmt.Fprintln(b, ":param pulumi.ResourceOptions opts: Options for the resource.")
 	if !argOverload {
 		for _, prop := range res.InputProperties {
-			mod.genPropDocstring(b, InitParamName(prop.Name), prop, true /*acceptMapping*/)
+			docRef := codegen.NewDocRef(codegen.DocRefTypeResourceInputProperty, res.Token, prop.Name)
+			mod.genPropDocstring(b, InitParamName(prop.Name), docRef, prop, true /*acceptMapping*/)
 		}
 	}
 
 	// printComment handles the prefix and triple quotes.
 	printComment(w, b.String(), "        ")
 }
 
+func renderComment(comment string, selfRef codegen.DocRef, filterExamples bool) string {
+	if comment == "" {
+		return ""
+	}
+	if filterExamples {
+		comment = codegen.FilterExamples(comment, "python")
+	}
+	return codegen.InterpretPulumiRefs(comment, func(ref codegen.DocRef) (string, bool) {
+		if ref.Token == "" {
+			return "", false
+		}
+		pyType := strings.ToLower(string(ref.Token.Module())) + "." + string(ref.Token.Name())
+		if ref.Property == "" {
+			return pyType, true
+		}
+		if ref.IsWithin(selfRef) {
+			return PyName(ref.Property), true
```

Comment:
I don't think just PyName is sufficient if properties also have language overrides set? Like you need to grab the resource schema and check that to get the right name.

Reply:
The module/resource paths needed the extra overrides, but the property names don't appear to have any special overrides supported:

https://github.com/pulumi/pulumi/blob/0d43ab628bb20cac609fdef6131b603d66aa98e1/pkg/codegen/python/gen.go#L1658-L1670